src/pages/diff/[slug].tsx

```diff
@@ -156,8 +150,25 @@ const DiffPage: NextPage<{
           </div>
         </button>
         {isExpanded && (
-          <Diff viewType={viewType} diffType={type} hunks={hunks}>
-            {(hunks) => hunks.map(renderHunk)}
+          <Diff
```

Comment:
why did you remove the renderHunk method ?

Reply:
I've moved it so I could get:
- inherited types
- I could use index for keys (React was yelling at me for duplicate keys in the map)

Reply:
Alright, sounds good 👌